### DISCUSSION TITLE
Suggest to fix the shell command

### DISCUSSION THREAD

**Discussion by wangyong31893189** [nuxt/nuxt#16472] [Questions] [answered]
![image](https://user-images.githubusercontent.com/8412484/200008599-accfcbbe-3057-415b-b392-d7c98f851303.png)
The command 'code nuxt-app' whether should update to 'cd nuxt-app'

**Comment by pi0** [comment, marked as answer]
Hi dear @wangyong31893189. Actually it is not a typo. `code` is a command you can enable:

https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line

It does cd to directory and open VSCode there.

**Reply by wangyong31893189** [reply]
Got it
